### ISSUE TITLE
Mod request

### ISSUE THREAD

**Issue by ClassyH** [berendbro/CookiesRulezzzExpert#6]
**Mod name and link**
MmmMmmMmmMmm    https://minecraft.curseforge.com/projects/mmmmmmmmmmmm

**Is your feature request related to a problem or is it just an amazing mod? .**
Extremely cool and very small mod, can be used well in this pack because it has a great synergy with tinkers weapons and others.

**Describe the solution/defend your mod**
This is a small mod that adds exactly one item: a Test Dummy.

The dummy can be placed in the world, where it'll stand silently, judging your every action. It'll wiggle funny if you hit it, screaming large numbers of damage at you. It can be dressed up to look even more awesome. This will please the dummy, making the numbers smaller.

TL;DR: Testdummies show damage dealt to them and can be equipped with armor 

**Additional context**
![Knipsel](https://user-images.githubusercontent.com/37928757/59549455-ffed8f00-8f5d-11e9-8d07-0a10f9d028b1.PNG)

**Comment by berendbro** [comment]
Seems useful to me, i'll check and test it in a few min
